### ISSUE TITLE
Narrator does not associate dropdown or combobox label with new selection

### ISSUE THREAD

**Issue by lherrle** [OfficeDev/office-ui-fabric-react#9896]
This issue has come up in an accessibility bug for Dynamics 365 PSA.

## Codepen demonstrating the issue
 
This behavior repros on the fabric ui page at https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown and https://developer.microsoft.com/en-us/fabric#/controls/web/combobox.

## Actual behavior:
After selecting a new value for the combo box or dropdown, the label (or aria-label) associated with the component is not read.

## Expected behavior:
After selecting a new value, the label or aria-label should be read in order to provide context for the selection.

## Documentation describing expected behavior
See https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/1320056/

**Comment by ecraig12345** [comment]
I can't reproduce this with Edge+Narrator (the primary combination we support). Can you be more specific about the steps you're using to reproduce this issue?
